### PR TITLE
updated the applications descriptions

### PR DESCRIPTION
**Pull request by qayyum-ibrahim**
If there is anything else to update text me.

### CODE REVIEW

**Comment by theharryszn** [comment]
Nice work @TEMIYORES  👍🏾👍🏾

Kindly remove the **Used to** prefix from all the descriptions.

Like this

From: 

```js
{
    description: "Used to display Images"
}
```

To:
```js
{
    description: "View Images"
}
```

Thanks

**Comment by qayyum-ibrahim** [comment]
roger

File: src/apps/Settings/index.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Home from "./Home";
 export const Settings: App = {
   name: "Settings",
   icon: require("./assets/logo.png"),
-  description: "User Preferences",
+  description: "Used to change user preferences",
```
Comment (theharryszn) [inline src/apps/Settings/index.ts]: Change this to **Manage User Preferences**

File: src/apps/Space/index.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Home from "./Home";
 export const Space: App = {
   name: "Space",
   icon: require("./assets/logo.png"),
-  description: "Explore the Infinite sWeb",
+  description: "Used to access information on the internet",
```
Comment (theharryszn) [inline src/apps/Space/index.ts]: Change this to **Explore the infinite Web**

**Comment by theharryszn** [comment]
Good work @TEMIYORES 👍🏾

**Comment by qayyum-ibrahim** [comment]
> Good work @TEMIYORES 👍🏾

Thank you boss

**Comment by Davidson015** [comment]
Well Done sire @TEMIYORES🙇‍♂️. From: TEMIYORESSent: Saturday, 18 June 2022 4:32 pmTo: asterix-oss/on-osCc: SubscribedSubject: [asterix-oss/on-os] updated the applications descriptions (PR #2) If there is anything else to update text me.You can view, comment on, or merge this pull request online at:  https://github.com/asterix-oss/on-os/pull/2Commit Summarye455ec6 updated the applications descriptionsFile Changes (10 files)M src/apps/Calculator/index.ts (2) M src/apps/Calendar/index.ts (2) M src/apps/Explorer/index.ts (2) M src/apps/Gallery/index.ts (2) M src/apps/Music/index.ts (2) M src/apps/Notes/index.ts (2) M src/apps/Settings/index.ts (2) M src/apps/Space/index.ts (2) M src/apps/Videos/index.ts (2) M yarn.lock (5941) Patch Links:https://github.com/asterix-oss/on-os/pull/2.patchhttps://github.com/asterix-oss/on-os/pull/2.diff—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by Davidson015** [comment]
Well-Done Sire.🙇‍♂️

**Comment by Davidson015** [comment]
Well-Done Sire @TEMIYORES.🙇‍♂️

**Pull request merged**